runtime/doc/eval.txt

```diff
@@ -2476,6 +2477,20 @@ call({func}, {arglist} [, {dict}])			*call()* *E699*
 		{dict} is for functions with the "dict" attribute.  It will be
 		used to set the local variable "self". |Dictionary-function|
 
+capture({command})					*capture()*
+		Capture output of {command}.
+		{command} is a |String|, it is passed as {command}.
+		{command} is a |List|, return all results of {command} of each
+		items.
+		Examples: >
+			let a = capture('echo "foo"')
+			echo a
+<			foo >
+			let a = capture(['echo "foo"', 'echo "bar"'])
+			echo a
+<			foobar  >
```

Comment:
In your test, this would be "\nfoo\nbar". Is this an oversight, or did I just miss something?